libcloud/container/drivers/lxd.py

```diff
@@ -971,6 +1073,25 @@ def list_images(self):
             images.append(self.ex_get_image(fingerprint=fingerprint))
         return images
 
+    def ex_has_image(self, alias):
+        """
+        Helper function. Returns true is the image with the
+        given alias exits on the host
+
+        :param alias: the image alias
+        :type  alias: ``str``
+
+        :rtype: boolean
+        """
+
+        # get all the images existing on the host
+        images = self.list_images()
+
+        for img in images:
```

Comment:
I assume there is no more efficient way of doing that (aka filter on the server side) and we need to do late filtering locally?

Reply:
There is after all, this was overlooked, thank you for bringing it to my attention.